setup.py

```diff
@@ -23,6 +23,7 @@
       py_modules=['imagesize'],
       test_suite='test',
       python_requires=">=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*",
+      install_requires=['requests',],
```

Comment:
We only need requests for tests, right? So that should probably be in `tests_require` rather than install_requires

Reply:
I switched to `urllib.request` and got rid of `requests` altogether. Also, `hasattr(filepath, "read")` is changed to `isinstance(filepath, io.BytesIO)` for better checking of a file-like object in https://github.com/shibukawa/imagesize_py/pull/46/commits/27569369af996e4d32d451e6207ae60902a69ffb

Reply:
Ah, yes, that's probably a good change. Something supporting read() might not also support seek() and such, after all. Though the idea of duck-typing it is nice